hr_payroll_expense/models/hr_contract.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# © 2016 Coninckx David (Open Net Sarl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+
+
+class HrContract(models.Model):
+    _inherit = 'hr.contract'
+
+    reimbursement = fields.Float(string='Reimbursement',
+                                 compute='_comp_reimbursement')

```

Comment:
I don't think it's a good idea because an employee can have multiple payslip in draft at the same time.
If this was placed in payslip how can we choose the payslip where to put the reimbursement.